.docker/web/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8
+
+ENV PYTHONUNBUFFERED 1
+
+RUN mkdir /code
+WORKDIR /code
+
+RUN pip install --upgrade pip
+
+ADD requirements.txt .
```

Comment:
Are we going to be keeping this and top-level requirements in sync? Wouldn't it be better to just add the top-level one?